doc/pages/JenkinFeatures.md

```diff
@@ -0,0 +1,145 @@
+| `KEEP_REPORTDIR`           | Optional, useful for showing NUnit output details with the JUnit Jenkins plugin.                                                      |
+| `ARTIFACTORY_SERVER`       | Optional, send artifacts to an Artifactory server if it's configured.                                                                 |
+| `ARTIFACTORY_REPO`         | Optional, use in conjunction with ARTIFACTORY_SERVER.                                                                                  |
+| `ARTIFACTORY_ROOT_DIR`     | Optional, use in conjunction with ARTIFACTORY_SERVER.                                                                                  |
+| `CLOUD_PROVIDER`           | Optional, if set, Jenkins jobs may try to spin up dynamic agents to run tests on if all real nodes are in use.                        |
+| `USE_TESTENV_PROPERTIES`   | Optional, boolean, use the values provided in the testenv.properties file to pin to particular versions of test material.            |
+| `RERUN_ITERATIONS`         | Optional, if set, indicates that when test targets fail, they are to be rerun this many times.                                         |
+| `RELATED_NODES`            | Setting the client machine label for use in client/server testing.                                                                     |
+
+## Grinder Etiquette
+- When possible, avoid running top-level targets (all of sanity.openjdk tests, etc., since we have nightly runs for those), or runs with 100x+ ITERATIONS.
+These can take hours and will block the regular testing, especially on platforms where we have few machines.
+- Do not mark Grinder runs as “keep forever,” there is a limited amount of space on Jenkins master.
+Download any artifacts you need from your run as soon as it finishes to attach to issues, etc., and consider the Jenkins job transient.
+- When reporting failures in issues, understand that Grinder jobs are transient (not kept for long on Jenkins).
+Sharing a link to the Grinder is only useful for a short time.
+Share the "Rerun in Grinder" link which has details of the impl/version/test target, output statistics & test artifacts in the issue; those are more concrete items when reproducing failures.
+For openjdk test failures, here are some useful triage [instructions](https://github.com/adoptium/aqa-tests/wiki/Guidance-for-Creating-OpenJDK-Test-Defects).
+- Proactively delete Grinder runs that you do not need (setup fails or mistake in target, etc.).
+We keep the last 50 runs, and if you delete your unneeded runs immediately, other jobs will stay a little bit longer, giving others more time to grab artifacts.
+
+**Note**: The above documentation provides a comprehensive reference for configuring Jenkins Test Jobs for AQAvit Grinder testing at ci.adoptium.net and OpenJ9 Jenkins server.
+Make sure to follow the guidelines and etiquette to ensure efficient and effective testing processes.
```

Comment:
This section can be removed from the reference documentation.  It can go into a different doc that is not talking about the details of how the Jenkins pipeline function, but rather a request for how the community is expected to behave.